app/models/report.rb

```diff
@@ -29,6 +29,8 @@ def initialize(type)
     @modes = [:table, :chart]
     @prev_data = nil
     @dates_filtering = true
+    @filter_options = nil
+    @filter = nil
```

Comment:
maybe it should be plural ?

Reply:
I think it would be okay to make it plural, though, the value of `filter` is a string, and not an array / collection / enumerable. So it is a single value, from correctness point of view, it is a singular value.